docs/faq/faq.md

```diff
@@ -11,14 +11,15 @@ You need ethernet LAN connection with the FPGA board to connect to it via ssh.
 The IP of your FPGA board can be found using the `ifconfig` command.
```

Comment:
I think "IP address" would be better in this sentence.

Reply:
I agree